backend/src/v5/services/mailer/templates/html/baseTemplate.html

```diff
@@ -220,7 +223,7 @@
 			</div>
 
 			<p class="footNotes">
-				Copyright &#0064; 2022 Asite 3D Repo Ltd. all rights reserved.
+				Copyright &#0064; 2025 Asite 3D Repo Ltd. all rights reserved.
```

Comment:
shouldn't be updating the year
![image](https://github.com/user-attachments/assets/a10f5ba6-711f-4447-b324-e7df8b39ab30)